src/SQLChecks/Functions/Public/Get-AutoStartupSQLServices.ps1

```diff
@@ -13,15 +13,18 @@ Function Get-AutoStartupSQLServices {
     }
 
     $query = @"
-    select  count(*) as [Count]
-    from    sys.dm_server_services
-    where   (
-                servicename like 'SQL Server (%)%'
-        or    servicename like 'SQL Server Agent%'
-            )
-    and     startup_type_desc <> 'Automatic';
+select servicename
+from sys.dm_server_services
+where
+(
+    servicename like 'SQL Server (%)%'
+    or servicename like 'SQL Server Agent%'
+)
+and startup_type_desc <> 'Automatic';
 "@
 
-    Invoke-Sqlcmd -ServerInstance $serverInstance -query $query -Database master
-    
+    (Invoke-Sqlcmd -ServerInstance $serverInstance -Query $query -Database master) |
+        ForEach-Object {
+            [string]$_.servicename
+        }
```

Comment:
I like the explicit connection to _master_. The intent is more evident than relying on hidden defaults. 

I'll replace the `foreach `with the property.

Comment-based help should be a separate endeavour. Now it would inflate the scope of the change as well as standout as the only function with a (partial) documentation